api/candidateApiClientModule.ts

```diff
@@ -0,0 +1,83 @@
+import getConfig from 'next/config';
+import fetch from "node-fetch";
+import {GetServerSideProps} from "next";
+import {ServerResponse} from "http";
+import {ParsedUrlQuery} from "querystring";
+
+const {publicRuntimeConfig} = getConfig();
+const baseUrl = publicRuntimeConfig.API_URL;
+
+export async function checkToken(token: string): Promise<boolean> {
+    const response = await fetch(`${baseUrl}/sessions/${token}`);
+    return response.ok;}
+
+export async function assertTokenIsValid(query: ParsedUrlQuery, response: ServerResponse): Promise<string> {
```

Comment:
I still think this doesn't need to return anything.
Also could you move it back to where it was? Its not really part of the API client (because it doesn't directly call the API)